### DISCUSSION TITLE
Is it based on mpc-hc or written from scratch?

### DISCUSSION THREAD

**Discussion by RomanSh-shell** [mpc-qt/mpc-qt#266] [Q&A]
I see that there is not much difference for the user, but it is very interesting to find out

**Comment by tbertels** [comment]
It's written from scratch, see https://github.com/mpc-qt/mpc-qt/discussions/155#discussioncomment-10521342 and https://github.com/mpc-qt/mpc-qt/discussions/155#discussioncomment-10526667.
